lib/activerecord-import/import.rb

```diff
@@ -225,13 +225,31 @@ def support_setting_primary_key_of_imported_objects?
     #
     # ====  Using A Hash
     #
-    # The :on_duplicate_key_update option can be a hash of column name
+    # The :on_duplicate_key_update option can be a hash of column names
     # to model attribute name mappings. This gives you finer grained
     # control over what fields are updated with what attributes on your
     # model. Below is an example:
     #
     #   BlogPost.import columns, attributes, :on_duplicate_key_update=>{ :title => :title }
     #
+    # == On Duplicate Key Update (Postgres 9.5+)
+    #
+    # The :on_duplicate_key_update option is a hash with :key and
+    # :columns attributes. Below is an example:
+    #
+    #   BlogPost.import columns, values, :on_duplicate_key_update=>{ :key => :id, :columns => [ :date_modified, :content, :author ] }
```

Comment:
Could you explain what you mean by passing `key` to use existing index? Postgres has the ability to determine what index maps to a list column names. The Postgres docs say that a constraint name can be explicitly specified, but is not recommended. Should I look into supporting that option?